tests/migration_test.go

```diff
@@ -1084,6 +1075,7 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 			})
 			It("[test_id:2227]should abort a vmi migration without progress", func() {
 				tests.SkipIpv6TestsFailingAfterVmImageUpgradeToFc32()
+				tests.SkipStressTestIfRunnigOnKindInfra()
```

Comment:
Can you please remove one of the skips? I guess the 'SkipMigrationTestIfRunnigOnKindInfra' should be removed.

Reply:
there are 2 issues that need to be fixed, they are independent 
this is why there are 2 lines, is it ok ?

Reply:
I added the 'SkipIpv6TestsFailingAfterVmImageUpgradeToFc32' since I thought it was caused by the upgrade. But since I see now you had this skip even before the upgrade, I understand now the test is flaky, unrelated to the upgrade.
I'm afraid that if the pressure issue would be fixed before the upgrade, the test will still be skipped (although there is a chance the upgrade didn't harm the test).

But since we don't actually know, you can keep both of them, up to you.